src/config_magic.c

```diff
@@ -70,6 +70,7 @@ const struct NamedCommand magic_shot_commands[] = {
   {"AREADAMAGE",      6},
   {"SPEED",           7},
   {"PROPERTIES",      8},
+  {"PUSHONHIT",       9},
```

Comment:
I don't know why all the other changes we already committed yesterday are listed again. I checked the main branch, and they are already in.

Reply:
weird